File: apiteddymo/app/Models/Portfolio.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Concerns\HasUuids;
use Illuminate\Database\Eloquent\Model;

class Portfolio extends Model
{
    use HasUuids;
    
    protected $fillable = [
        'id', 'title', 'summary', 'description', 'technologies', 'link', 'imageUrl','is_deleted'
    ];

    protected $casts = [
        'technologies' => 'array', // Cast to array for JSON field
    ];
}

File: DotnetAPI/Models/Portfolio.cs
namespace DotnetAPI.Models
{
    public class Portfolio
    {
        public Guid Id { get; set; }
        public required string Title { get; set; }
        public required string Summary { get; set; }
        public required string Description { get; set; }
        public required string Technologies { get; set; }
        public string? Link { get; set; }
        public string? ImageUrl { get; set; }
        public bool IsActive { get; set; }
        public DateTime? DeletedDate { get; set; }
    }
}
